server/movies/routes.py

```diff
@@ -13,18 +13,21 @@ def invalid_api_usage(e):
     return jsonify(e.serialize()), e.status_code
 
 
-@movies_bp.route('/api/watchlist', methods=["GET", "POST"])
```

Comment:
this route handler was getting big, so I separated it into separate handlers for each method